src/main/java/mezz/jei/gui/overlay/IngredientListOverlay.java

```diff
@@ -304,10 +304,18 @@ public boolean onGlobalKeyPressed(int eventKey) {
 				Config.toggleCheatItemsEnabled();
 				return true;
 			}
+			if(KeyBindings.toggleEditMode.isActiveAndMatches(eventKey)) {
+				Config.toggleEditModeEnabled();
+				return true;
+			}
 			if (KeyBindings.focusSearch.isActiveAndMatches(eventKey)) {
 				setKeyboardFocus(true);
 				return true;
 			}
+			if(KeyBindings.toggleEditMode.isActiveAndMatches(eventKey)) {
```

Comment:
I think this second check is not needed. If it is, it should have a explanatory comment.

Reply:
Fixed in [18243e31](https://github.com/mezz/JustEnoughItems/pull/1530/commits/18243e31cfb436d520ca9bceb2fe57c84aee58e7)